server/test/api/projects.js

```diff
@@ -55,6 +56,34 @@ describe('The /projects API endpoint', function () {
       .catch(helpers.handler);
   });
 
+  it('GET /projects/ ? COMPLETE = 1 returns a complete List of project  ', function () { 
+    return agent.get('/projects?complete=1')
+      .then(function (result) { 
+        expect(result).to.have.status(200);
+        expect(result).to.be.json;
+      })
+      .catch(helpers.handler);
+  });
+
+  it('GET /projects/ ? UNLOCKED = 1 returns a complete List of unlocked projects  ', function () { 
+    return agent.get('/projects?unlocked=1')
```

Comment:
Can you write a test to verify that only unlocked projects get returned from this query?  How many properties do you expect them to have?